agent-install/agent-install.sh

```diff
@@ -1696,13 +1711,23 @@ function get_arch() {
         if is_debian_variant; then
             dpkg --print-architecture
         elif is_redhat_variant; then
-            uname -m   # x86_64 or aarch64 (i think)
+            uname -m
         fi
     elif is_macos; then
         uname -m   # e.g. x86_64. We don't currently use ARCH on macos
     fi
 }
 
+# Returns hardware architecture for Docker image names
+function get_image_arch() {
+    local pkg_arch=${ARCH:-$(get_arch)}
+    if [[ $pkg_arch =~ ^(x86_64|amd64)$ ]]; then
+        echo 'amd64'
+    elif [[ $pkg_arch == 'ppc64le' ]]; then
+        echo 'ppc64le'
+    fi
```

Comment:
The problem with this is if a developer adds an arch to the variables at the top of the script, they need to know to also come here and add it. I'd rather have this function only make the modifications needed, instead of needing to be a complete list. Like this:

```
    local image_arch=${ARCH:-$(get_arch)}
    if [[ $image_arch == 'x86_64' ]]; then
        image_arch='amd64'
    elif [[ $image_arch == 'aarch64' ]]; then
        image_arch='arm64'
    fi
    echo $image_arch
```

Reply:
Ok, fixed that